unison-src/transcripts/cycle-update-3.output.md

```diff
@@ -51,20 +51,30 @@ ping = 3
 
 ```
 ```ucm
-.> update.old
+.> update
 
-  ⍟ I've updated these names to your new definition:
-  
-    ping : Nat
+  Okay, I'm searching the branch for code that needs to be
+  updated...
 
-.> view ping pong
+  That's done. Now I'm making sure everything typechecks...
 
-  ping : Nat
-  ping = 3
-  
-  pong : 'Nat
-  pong _ =
-    use Nat +
-    !#4t465jk908.1 + 2
+  Typechecking failed. I've updated your scratch file with the
+  definitions that need fixing. Once the file is compiling, try
+  `update` again.
+
+```
+```unison:added-by-ucm scratch.u
```

Comment:
Just double checking @mitchellwrosen  is this the new expected behaviour from new `update` w/r to cycles?

Reply:
@ChrisPenner Is the question wrt cycles, "why didn't it just silently succeed"?

Reply:
This transcript looks right to me, just wanting to confirm with @mitchellwrosen  that this transcript is correct w/r to new update; 

That is, if you update only part of a cycle it should bring the rest of the cycle into scope.

Reply:
Yeah, that's correct @ChrisPenner